src/backend/fs.rs

```diff
@@ -37,26 +47,22 @@ pub async fn write_file(pk: Secp256k1PubKey, file_bytes: &[u8]) -> Result<Blake3
     encoded_segments.push(last_segment);
 
     trace!("Get eight storage volumes from config");
-    if SYS_CFG.volumes.len() != 8 {
+    if SYS_CFG.volumes.len() != FEC_M {
```

Comment:
Is there ever a case where SYS_CFG.volumes.len() ... "would not be 8"?

Reply:
Well, it's config, so it could possibly occur if there's a misconfiguration. Maybe this should happen in config, but that can be a chore done in a separate PR.